frontend/src/components/elements/DeckGlJsonChart/DeckGlJsonChart.tsx

```diff
@@ -137,6 +137,19 @@ export class DeckGlJsonChart extends PureComponent<PropsWithHeight, State> {
   static getDeckObject = (props: PropsWithHeight): DeckObject => {
     const { element, width, height } = props
     const json = JSON.parse(element.json)
+    let theme
```

Comment:
you can use `const` here since it is only set one time and move the initialization down to line `146`